pkg/scannerv4/mappers/mappers.go

```diff
@@ -745,20 +693,25 @@ func cvssVector(cvssVector string) (severityValues, error) {
 		severity: cvssVector,
 	}
 
-	// Guess the CVSS version by prefix.
-	if strings.HasPrefix(cvssVector, "CVSS:3") {
-		if _, err := cvss3.VectorFromString(cvssVector); err != nil {
-			return values, fmt.Errorf("parsing CVSS v3 vector: %w", err)
+	// TODO: add CVSS v4 support.
+	switch {
+	case strings.HasPrefix(cvssVector, `CVSS:3.0`), strings.HasPrefix(cvssVector, `CVSS:3.1`):
```

Comment:
Why the condition change here? were there `CVSS:3*` patterns causing breakage of sorts?

Reply:
I originally was going to replace `github.com/facebookincubator/nvdtools/cvss*` usage with https://github.com/quay/claircore/tree/main/toolkit/types/cvss but that API doesn't offer just the base score (the score you get is based on all vectors, not just the base, and I didn't want to modify the vector to just get the base metrics when I can just keep using this other library). The claircore library does this type of check for CVSS version https://github.com/quay/claircore/blob/toolkit/v1.2.4/toolkit/types/cvss/cvss.go#L96 so I figured I'd mirror that